config.json

```diff
@@ -1850,26 +1850,32 @@
   ],
   "key_features": [
     {
+      "icon": "fast",
       "title": "Performance",
       "content": "Rust is blazingly fast and memory-efficient; it has no runtime or garbage collector."
     },
     {
+      "icon": "stable",
       "title": "Reliability",
       "content": "Rust’s rich type system and ownership model guarantee memory-safety and thread-safety."
     },
     {
+      "icon": "productive",
       "title": "Productivity",
       "content": "Rust has great documentation, a friendly compiler with useful error messages, and top-notch tooling."
     },
     {
+      "icon": "tooling",
       "title": "Cargo",
       "content": "Rust's package manager and build tool is best-in-class."
     },
     {
+      "icon": "cross-platform",
```

Comment:
![`cross-platform`](https://raw.githubusercontent.com/exercism/website-icons/main/key-features/cross-platform.svg) `cross-platform`

You may prefer e.g.

![`portable`](https://raw.githubusercontent.com/exercism/website-icons/main/key-features/portable.svg) `portable`